src/metabase/models/persisted_info.clj

```diff
@@ -74,14 +75,30 @@
        (take 10)
        (apply str)))
 
+(defenterprise refreshable-states
+  "States of `persisted_info` records which can be refreshed.
+
+   Without the :cache-granular-controls feature flag, any models with state=off should be treated as if they were
+   in state=creating (i.e. they should be refreshable, and not prunable). This situation is only possible if the
+   instance previously had the :cache-granular-controls feature flag, then downgraded to not have it."
+  metabase-enterprise.advanced-config.caching
```

Comment:
This still took me a minute to understand why `:off` belongs here, I assumed that `:off should be pruned, but really what this is doing is turning any `off` models back `on` because without `cache-granular-controls` all models should be on.

Reply:
Yes, you figured it out. I've updated the docstring to hopefully be clearer about this for the next person that walks by this code
[Update docstring to hopefully be clearer](https://github.com/metabase/metabase/pull/32641/commits/9405283ed2f4b0191f210895aa4cea32fd71450d)